Tchap/Managers/Invite/InviteService.swift

```diff
@@ -120,12 +111,16 @@ final class InviteService: InviteServiceType {
     // Check whether a Tchap account has been created for this email. The closure returns a nil identifier when no account exists.
     private func discoverUser(with email: String, completion: @escaping (MXResponse<InviteServiceDiscoverUserResult>) -> Void) {
         let pid = MX3PID(medium: .email, address: email)
-        _ = self.identityService.lookup3PIDs([pid]) { response in
+        guard let identityService = self.session.identityService else {
```

Comment:
as a good practice, we would move the guard let as the first position of the block
(let pid... should be archived after the guard let)

Reply:
I moved this on the last commit.